pkgs/development/python-modules/mpris-server/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, emoji
+, pydbus
+, pygobject3
+, unidecode
+, setuptools
+,
+}:
+buildPythonPackage rec {
+  pname = "mpris-server";
```

```suggestion
  pname = "mpris_server";
```

naming the package the same name as the python module will make searching and including it easier for future users.

---

This is the only suggestion i disagree with, the name is mpris-server on PyPI: https://pypi.org/project/mpris-server/

---

named mpris_server on authors github page. https://github.com/alexdelorenzo/mpris_server I don't know why the name is different on PyPI in the docs it is always `mpris_server`.

`pip install mpris_server` works but `nix-shell -p python3.withPackages(p: [p.mpris_server])` will not.

---

Okay, so I should rename it, will do

---

Done